components/infobox/wikis/smite/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,65 @@
+---
+-- @Liquipedia
+-- wiki=smite
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Region = require('Module:Region')
+local Role = require('Module:Role')
```

Comment:
Either make it a standard Infobox Extension, and drop it and put the login in the custom

Reply:
that would mean this required a base infobox edit too? (if this refers to Class and Lua calls should no longer be in the custom infobox league, while Region has been removed from this one but Role is kept as wiki custom)

Reply:
It's a reference to Role. Github display 3 lines above for single line comments for context. If it's a multi-line comment github always displays the lines numbers
![image](https://github.com/Liquipedia/Lua-Modules/assets/3426850/f1b86755-f028-4b89-88c2-62edfb0d363e)

Reply:
I see, if thats the case then, I might not need to adjust anymore then? 

since Module:Role don't exist in commons (only as [wiki locals](https://liquipedia.net/smite/Module:Role)) if I moved `local Role = require('Module:Role')` into the base person infobox over the custom, wouldnt it caused a missing module error?

Reply:
**imo** it would be good to standardize the role stuff
but **imo** oos for this PR

Reply:
Role stuff doesn't need to be standardized, but I do not like Module:Role. Move that logic into the Custom